aws/table_aws_route53_domains.go

```diff
@@ -0,0 +1,180 @@
+		Description: "AWS Route53 Domains",
+		Get: &plugin.GetConfig{
+			KeyColumns: plugin.SingleColumn("name"),
+			Hydrate:    getRoute53Domain,
+		},
+		List: &plugin.ListConfig{
+			Hydrate: listRoute53Domains,
+		},
+		Columns: awsColumns([]*plugin.Column{
+			{
+				Name:        "name",
+				Description: "The name of the domain.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("DomainName"),
+			},
+			{
+				Name:        "auto_renew",
+				Description: "Indicates whether the domain is automatically renewed upon expiration.",
+				Type:        proto.ColumnType_BOOL,
+			},
+			{
+				Name:        "expiry",
+				Description: "Expiration date of the domain in Unix time format and Coordinated Universal Time (UTC).",
+				Type:        proto.ColumnType_TIMESTAMP,
+			},
+			{
+				Name:        "transfer_lock",
+				Description: "Indicates whether a domain is locked from unauthorized transfer to another party.",
+				Type:        proto.ColumnType_BOOL,
+			},
+			{
+				Name:        "tags_src",
+				Description: "A list of tags assigned to the parameter.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getAwsRoute53DomainTags,
+				Transform:   transform.FromField("TagList"),
+			},
+
+			// Standard columns for all tables
+			{
+				Name:        "title",
+				Description: resourceInterfaceDescription("title"),
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("DomainName"),
+			},
+			{
+				Name:        "tags",
+				Description: resourceInterfaceDescription("tags"),
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getAwsRoute53DomainTags,
+				Transform:   transform.FromField("TagList").Transform(route53DomainsTagListToTurbotTags),
+			},
+			{
+				Name:        "akas",
+				Description: resourceInterfaceDescription("akas"),
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getAwsRoute53DomainAkas,
+				Transform:   transform.FromValue(),
+			},
+		}),
+	}
+}
+
+//// LIST FUNCTION
+
+func listRoute53Domains(ctx context.Context, d *plugin.QueryData, _ *plugin.HydrateData) (interface{}, error) {
+	// Create session
+	svc, err := Route53DomainsService(ctx, d, "us-east-1")
+	if err != nil {
+		return nil, err
+	}
+
+	// List call
+	err = svc.ListDomainsPages(
+		&route53domains.ListDomainsInput{},
+		func(page *route53domains.ListDomainsOutput, isLast bool) bool {
+			for _, domain := range page.Domains {
+				d.StreamListItem(ctx, domain)
+			}
+			return !isLast
+		},
+	)
+	return nil, err
+}
+
+////  HYDRATE FUNCTIONS
+
+func getRoute53Domain(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	logger := plugin.Logger(ctx)
+	logger.Trace("getRoute53Domain")
+
+	// Create session
+	svc, err := Route53DomainsService(ctx, d, "us-east-1")
+	if err != nil {
+		return nil, err
+	}
+
+	// Build the params
+	params := &route53domains.ListDomainsInput{}
+
+	// Get call
+	data, err := svc.ListDomains(params)
```

Comment:
Can we use `GetDomainDetail` API for GET?
https://docs.aws.amazon.com/Route53/latest/APIReference/API_domains_GetDomainDetail.html